src/lib.rs

```diff
@@ -62,6 +63,10 @@ pub trait Cache: Send + Sync {
         ttl: NonZeroU64,
     ) -> anyhow::Result<()>;
     async fn get<'a>(&'a self, key: &'a Self::Key) -> anyhow::Result<Option<Self::Value>>;
+
+    fn hit_rate(&self) -> Option<f64> {
+        unimplemented!()
```

Comment:
Is this work in progress?

Reply:
I had implemented `hit_rate` only for InMemoryCache (cuz it has builtin support) and for other instances marked this as unimplemented by default i.e. until we'll add trace support to cloudflare and others

Reply:
I think it would be safer to return a hit_rate of `0` in those cases. Current implementation would result in some embarrasing and hard to debug wasm issues.